sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -130,105 +132,126 @@ def analyze_text():
         if e.error is not None:
```

Comment:
This can't be None if you response error JSON follows Azure guidelines

Reply:
@lmazuel I've replaced "if xxx is not None" with "if xxx", but if I remove the None check totally, the MyPy and Pyright will report errors, like below:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=2790574&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=e3f8f308-6e5f-58aa-3a2a-fe425958de32
![image](https://github.com/Azure/azure-sdk-for-python/assets/50131889/7f50be3e-602e-49fc-895a-a87d1f390858)
https://dev.azure.com/azure-sdk/public/_build/results?buildId=2790574&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=56d4e2e6-c43b-527c-bad7-234ebe7429dd
![image](https://github.com/Azure/azure-sdk-for-python/assets/50131889/8c617b27-4e25-4364-b5d8-ce25a3c246fc)
I will keep the None check to pass CI, maybe you could consider removing these checks later.